src/accelerate/utils/dataclasses.py

```diff
@@ -915,6 +930,19 @@ class GradientAccumulationPlugin(KwargsHandler):
 class TorchDynamoPlugin(KwargsHandler):
     """
     This plugin is used to compile a model with PyTorch 2.0
+
+    Args:
+        backend (`DynamoBackend`, defaults to `None`):
```

Comment:
Can you help me fill this one out @muellerzr?

Reply:
```suggestion
        backend (`DynamoBackend`, defaults to `None`):
        	A valid dynamo backend. See https://pytorch.org/docs/stable/torch.compiler.html for more details
```